packages/core/package.json

```diff
@@ -120,16 +127,18 @@
     "http-status": "^1.7.0",
     "jaro-winkler": "^0.2.8",
     "json-stringify-safe": "^5.0.1",
+    "langchain": "0.3.6",
     "lodash": "^4.17.21",
     "pg": "^8.11.3",
     "pkijs": "^3.0.16",
-    "playwright": "^1.39.0",
+    "playwright": "1.39.0",
```

Comment:
If we're not longer fixing the version, should we simply update the version to the new one?

Reply:
I don't understand what you mean. We are fixing the version. TBH, I don't remember why from all the fixes/loops we had to go through. Could try to leave the caret there and see if things work.

I know that we fixed some libs because we're importing code from langchain, and so we wanted to fix the version of those libs and the ones it imports.

Reply:
Oops -- had the caret purpose reversed 🤦